features/frameworks/greentea-client/greentea-client/test_env.h

```diff
@@ -28,6 +28,9 @@
 #endif
 
 #include <stdio.h>
+extern "C" {
```

Comment:
Ok if that is the practice. But we are making this header common for C and C++. 
For any odd reason we would want to switch to using .hpp or .hh extension for C++ headers than it would look odd.

I will update the code based on the suggestion.